src/main/java/com/csv/communitytrackerjava/service/ProjectServiceImpl.java

```diff
@@ -0,0 +1,97 @@
+package com.csv.communitytrackerjava.service;
+
+import com.csv.communitytrackerjava.dto.ProjectAddDTO;
+import com.csv.communitytrackerjava.dto.ProjectPayloadDTO;
+import com.csv.communitytrackerjava.dto.ProjectResponseDTO;
+import com.csv.communitytrackerjava.dto.ProjectUpdateDTO;
+import com.csv.communitytrackerjava.exception.ProjectCodeExistException;
+import com.csv.communitytrackerjava.exception.RecordNotFoundException;
+import com.csv.communitytrackerjava.mapper.ProjectMapper;
+import com.csv.communitytrackerjava.model.Project;
+import com.csv.communitytrackerjava.repository.ProjectRepository;
+import org.apache.commons.text.CaseUtils;
+
+import org.modelmapper.ModelMapper;
+import org.modelmapper.config.Configuration;
+import org.modelmapper.convention.MatchingStrategies;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class ProjectServiceImpl implements ProjectService {
+    @Autowired
+    ProjectRepository projectRepository;
+
+    @Autowired
+    ProjectMapper projectMapper;
+    
+    @Autowired
+    ModelMapper modelMapper;
+    
+    ProjectResponseDTO projectResponseDTO = new ProjectResponseDTO();
+
+    ProjectPayloadDTO payloadDTO = new ProjectPayloadDTO();
```

Comment:
same here avoid using private variables because if we have a concurrent transactions it may have an impact